exercism/bob/bob.src.js

```diff
@@ -0,0 +1,16 @@
+var Bob = function() {};
+
+Bob.prototype.hey = function(what) {
+    if (what === what.toUpperCase() && what.toUpperCase() != what.toLowerCase()){
```

Comment:
I would extract the condition of this if statement into its own function. It will make it much easier to read
```
if(isScreaming(what)){
   return "Whoa, chill out!"
}
```
Extracting logic into smaller functions is good programming practice, which makes the code readable, and much easier to maintain.